### PR TITLE
PCM-7968: can add multiple files with ctrl+click

### PR DESCRIPTION
**Pull request by jeudy100**

### CODE REVIEW

File: app/cases/controllers/attachLocalFile.js [hunk 1]
```diff
@@ -49,44 +49,45 @@ export default class AttachLocalFile {
         $scope.selectFile = function () {
             const minSize = 0;
```
Comment (rh-kgarg) [inline app/cases/controllers/attachLocalFile.js]: @jeudy100 If i select 3 files at first and then without uploading them i try to select 3 more files from the file explorer, will it override the previously selected files?

Comment (jeudy100) [reply app/cases/controllers/attachLocalFile.js]: @rh-kgarg It doesn't, it calls AttachmentsService.addNewAttachment for each file which just adds that to the end of the attachments array.

File: app/cases/views/attachLocalFile.jade [hunk 2]
```diff
@@ -11,7 +11,7 @@
     ng-click='getFile()',
     ng-disabled='CaseService.addingComment',
     translate='',
-    ng-hide='AttachmentsService.updatedAttachments.length > 0') Attach File
+    ng-hide='AttachmentsService.updatedAttachments.length > 0') Attach file(s)
```
Comment (jeudy100) [inline app/cases/views/attachLocalFile.jade]: @rh-kgarg I talked to @KrystalMoore16 and we decided to switch the "Attach file" button string to "Attach file(s)" to show that the user can now upload multiple files.

**Comment by anujsi** [comment]
Reviewed.

**Pull request merged**